contracts/script/deploy/verifiable/config/mainnet.config.toml

```diff
@@ -0,0 +1,65 @@
+### CORE ###
```

Comment:
Should this file be called `sepolia.config.toml`?

Reply:
Ah yeah, that is right. Thanks for the heads up